lib/app.js

```diff
@@ -28,9 +28,47 @@ app.get('/api/test', (req, res) => {
   });
 });
 
-app.get('/animals', async(req, res) => {
+
+
+//GET ROUTE
+app.get('/api/todos', async(req, res) => {
+  try {
+    const data = await client.query('SELECT * from todos WHERE owner_id = $1', [req.userId]);
+    
+    res.json(data.rows);
+  } catch(e) {
+    
+    res.status(500).json({ error: e.message });
+  }
+});
+
+
+
+//POST route
+app.post('/api/todos', async(req, res) => {
+  try {
+    const data = await client.query(`
+    INSERT INTO todos (todo, completed, owner_id) 
+    VALUES ($1, $2, $3) RETURNING *`, [req.body.todo, req.body.completed, req.userId]);
+    
+    res.json(data.rows);
+  } catch(e) {
+    
+    res.status(500).json({ error: e.message });
+  }
+});
+
+app.use(require('./middleware/error'));
+
+module.exports = app;
+
+
+
+//DELETE route
+app.delete('/api/todos/:id', async(req, res) => {
   try {
-    const data = await client.query('SELECT * from animals');
+    const data = await client.query(
+      'DELETE from todos where id=$1 RETURNING *', [req.params.id]);
```

Comment:
To make this more secure I would add on to the WHERE statement `AND owner_id = $2`